_js/api-ordered.json

```diff
@@ -6,7 +6,44 @@ permalink: "/api/data-ordered.json"
 	"api_version":"1.0.4-ordered",
 	"last_update_date":{{site.time | jsonify}},
 	"nicenames":{{site.data.nicenames | jsonify}},
-	"data":[{% for feature in site.features %}
+	"data":[
+		{% for feature in site.features %}
+		{
+			"slug":{{ feature.slug | jsonify }},
+			"title":{{ feature.title | jsonify }},
+			"description":{{ feature.description | jsonify }},
+			"url":{{ feature.url | prepend:site.url | jsonify }},
+			"category":{{ feature.category | jsonify }},
+			"tags":{{ feature.tags | jsonify }},
+			"keywords":{{ feature.keywords | jsonify }},
+			"last_test_date":{{ feature.last_test_date | jsonify }},
+			"test_url":{{ feature.test_url | prepend:site.url | jsonify }},
+			"test_results_url":{{ feature.test_results_url | jsonify }},
+			"stats":{
+				{%- for family in feature.stats -%}
+					{%- assign family-key = family | first -%}
+					{%- assign family-values = feature.stats[family-key] -%}
+					{{ family-key | jsonify }}:{
+						{%- for platform in family-values -%}
+							{%- assign platform-key = platform | first -%}
+							{%- assign platform-values = platform | last -%}
+							{{ platform-key | jsonify }}:[
+								{%- for version in platform-values -%}
+									{%- assign version-key = version | first -%}
+									{%- assign version-values = version | last -%}
+									{ {{ version-key | jsonify }}:{{ version-values | jsonify }} }
+									{%- unless forloop.last -%},{%- endunless -%}
+								{%- endfor -%}
+							]{%- unless forloop.last -%},{%- endunless -%}
+						{%- endfor -%}
+					}{%- unless forloop.last -%},{%- endunless -%}
+				{%- endfor -%}
+			},
+			"notes":{{ feature.notes | jsonify}},
+			"notes_by_num":{{ feature.notes_by_num | jsonify}}
+		},{% endfor %}
+		{% assign webfeatures = site.genfeatures | concat: site.features | sort:'id' | map_features -%}
```

Comment:
Can we copy what I did here?
https://github.com/WebView-CG/Compatibility-Data-Project/blob/main/_js/features.json

Seems like the logic is the same so doing a concat would be nice.

I also made a [plugin](https://github.com/WebView-CG/Compatibility-Data-Project/blob/main/_plugins/map_features.rb) for us which you're welcome to modify to get the fields we want. The benefit of using the mapping plugin is that we can then use jsonify which will minify these results.

https://caniwebview.com/assets/js/features.json vs https://caniwebview.com/api/data-ordered.json

Reply:
The plugin is used and I slightly modified it. I wasn't aware of concat in Jekyll, nifty. I will refactor this.